problems/sumEvenNumbers.js

```diff
@@ -9,8 +9,14 @@
 * ex: sumEvenNumbers([4, 3, 10])
 * returns: 14
 */
-function sumEvenNumbers() {
+function sumEvenNumbers(arr) {
+    let sum = arr.filter((el => el % 2 === 0))
+    if (sum[el]) {
+        sum += el
 
+    }
```

Comment:
you did a good job filtering through the array and getting only an array of even numbers, I would just assign it to a different variable and use the `sum` variable to sum up all the numbers. With that new after that came from the filter you would need to iterate through it with some sort of loop